### DISCUSSION TITLE
unable to deploy on ubuntu 64 bit on Raspberry Pi

### DISCUSSION THREAD

**Discussion by bklane2** [doitandbedone/ispyagentdvr-docker#205] [General]
Hi,
I have tried to deploy using multi-arch image but get "exec format error". I am really new to deploying containers so am I missing a parameter on my docker script below?

docker run -it -p 8090:8090 -p 3478:3478/udp -p 50000-50010:50000-50010/udp \
-v /appdata/ispyagentdvr/config/:/agent/Media/XML/ \
-v /appdata/ispyagentdvr/media/:/agent/Media/WebServerRoot/Media/ \
-v /appdata/ispyagentdvr/commands:/agent/Commands/ \
-v /mnt/camera_data/ispy:/mnt/camera_data/ispy \
--name ispyagentdvr doitandbedone/ispyagentdvr:multi-arch

**Comment by doitandbedone** [comment]
Unfortunately, multi arch is not fully finished yet. Please install
directly for RBP. The overhead may not be worth. Please follow instructions
in ispy website to install onto ARM64
